.github/workflows/ci.yml

```diff
@@ -32,8 +32,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
-        with:
-          submodules: true
+      - name: Checkout submodule
+        # Manually update submodules with --checkout because they are configured with update=none and will be skipped otherwise
+        run: git submodule update --recursive --init --force --checkout
```

Comment:
JFYI, I tested wheter setting `submodules: checkout` might be an undocumented feature where `--checkout` is added to the `submodule update` command, but no dice. It only seems to check for `== "true"`.